docs/quickstart.rst

```diff
@@ -227,6 +227,7 @@ Finally compile the project and deploy it:
 
 .. code-block:: sh
 
+    inmanta -vvv  project install
```

Comment:
This is already mentioned 6 lines earlier.

Reply:
How did I miss that. Removed this line.